components/NavBar.js

```diff
@@ -18,25 +17,12 @@ export default function NavBar (props) {
         <ul>
           <li>
             <NavLink href='/'>
-              Home
-            </NavLink>
-          </li>
-          <li>
-            <NavLink href='/faq'>
-              FAQ
+              Dashboard
             </NavLink>
           </li>
           <li>
-            <NavLink href='/about'>
-              About
-            </NavLink>
-          </li>
-          <li>
-            <NavLink
-              href='https://fdot-otp.ibi-transit.com/'
-              external
-            >
-              View RMCE
+            <NavLink href='/manage'>
+              Manage
```

Comment:
I think it'd be better to have just one link to home and then have a bunch of cards to select from at the home to do navigation.

Reply:
See #7.